doc/users/whats_new/path_simplification_updates.rst

```diff
@@ -0,0 +1,20 @@
+Path simplification updates
+---------------------------
+
+Lines will now be simplified according to the ``path.simplify`` and
+``path.simplify_threshold`` RC parameters, as long as other conditions
+such as not using dashed lines are met. Only the line segment portion
+of paths will be simplified. If you are also drawing markers and
+experiencing problems with rendering speed, you should consider using
+the ``markevery`` option to ``plot``.
+
+The simplification works by iteratively merging line segments
+into a single vector until the next line segment's perpendicular
+distance to the vector (measured in display-coordinate space)
+is greater than the ``path.simplify_threshold`` parameter. Thus, higher
+values of ``path.simplify_threshold`` result in quicker rendering times.
+If you are plotting just to explore data and not for publication quality,
+pixel perfect plots, then a value of ``1.0`` can be safely used. If you
+want to make sure your plot reflects your data *exactly*, then you should
+set ``path.simplify`` to false and/or ``path.simplify_threshold`` to ``0``.
+Currently, matplotlib defaults to a convervative value of ``1/9``.
```

Comment:
I think a version of this information should be in the core `path` documentation so that it is easier to stumble upon.